yardstick.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |gem|
   gem.executables      = %w[yardstick]
 
   gem.add_runtime_dependency('yard', '~> 0.8', '>= 0.8.7.2')
+  gem.add_runtime_dependency('concord', '~> 0.1')
```

Comment:
I'd go for `~> 0.1.x` as the version as below `1.0` I use left shifted semver and there is no expected public API stability cross `0.x`. I do not plan big changes and concord should have a `1.0` already but lets keep consistence.